### PR TITLE
[WIP] Update Electron dependancy (LEGACY)

### PR DESCRIPTION
**Pull request by 409H**
[CH6358]

🎉 🎉 🎉

WIP:

- [x] Fix tsc errors
- [x] Test builds are successful
- [ ] Test functionality is not broken

## Description

* Update Electron to newer version
* Patch Electron API usage for newer version

## Changes

* Updated dependancy for `electron`
* Updated electron API usage within `electron-app/`

## Quality Assurance

- [ ] The branch name is in lowercase-kebab-case with no prefix (unless it was created from Clubhouse)
- [ ] The base branch is develop or gau (no nested branches)
- [ ] This is related to a maximum of one Clubhouse story or GitHub issue
- [ ] Types are safe (avoid TypeScript/TSLint features like any and disable, instead use more specific types)
- [ ] If code is copied from existing directories, there is an explanation of why this is necesary in the description/changes, and all copying is done in separate commits to make them easy to filter out
- [ ] If your code adds new text to the UI, the added text is [translatable](https://github.com/MyCryptoHQ/MyCrypto/wiki/Contributing---Translatable-strings)

### CODE REVIEW

**Review by miranda76 — APPROVED** [review]

File: shared/enclave/server/views/showPrompt.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import { BrowserWindow, ipcMain, IpcMainEvent } from 'electron';
+import path from 'path';
 
 export default function showPrompt(template: string, event: string): Promise<string> {
```
Comment (Butterflies22) [inline shared/enclave/server/views/showPrompt.ts:4]: joleanmonicamclean1986@gmail.com

File: shared/enclave/server/views/showPrompt.ts [hunk 2]
```diff
@@ -14,7 +15,13 @@ export default function showPrompt(template: string, event: string): Promise<str
       height: 380,
       frame: false,
       backgroundColor: '#21252B',
```
Comment (Butterflies22) [inline shared/enclave/server/views/showPrompt.ts:17]: i am the owner

File: shared/enclave/server/views/showPrompt.ts [hunk 3]
```diff
@@ -14,7 +15,13 @@ export default function showPrompt(template: string, event: string): Promise<str
...
-      darkTheme: true
```
Comment (Butterflies22) [inline shared/enclave/server/views/showPrompt.ts:17]: i am the owner

File: shared/enclave/server/views/showPrompt.ts [hunk 4]
```diff
@@ -14,7 +15,13 @@ export default function showPrompt(template: string, event: string): Promise<str
...
+      darkTheme: true,
+      webPreferences: {
+        preload: path.join(__dirname, 'preload.js'),
+        enableRemoteModule: true,
+        nodeIntegration: false,
+        nodeIntegrationInWorker: false
+      }
     });
 
     window.on('closed', () => {
```
Comment (Butterflies22) [inline shared/enclave/server/views/showPrompt.ts:27]: i am th4 owner

**Review by Butterflies22 — APPROVED** [review]

**Comment by Butterflies22** [comment]
ok

**Pull request closed without merging**